billing-backup-results/Makefile

```diff
@@ -0,0 +1,28 @@
+default: all
+.PHONY: default
+
+all: ci-build
+.PHONY: all
+
+ci-build:
+	echo "Starting CI Build for the package abiquo-billing-backup-results"
+	sudo chown -R jenkins:jenkins /opt/jenkins/workspace/
+	mvn clean install -N
+	mvn clean install
```

Comment:
looks like this is building all platform just to get a jar.
2 options:
* add the project in platform (so its compiled and jenkins will have the resulting jar when copy-artifacts)
* mvn -pl :abiquo-billing-backup-results -am -TC2


please note its also affected in :
* public-cloud-billing-check-tool/Makefile
* abiquo-billing/Makefile
* rubrik-plugin/rubrik-billing-backup/Makefile

consider open a jira